README.md

```diff
@@ -95,8 +95,15 @@ The parameter accepted by the script are documented in the following.
 
 ##### Root Parameters
 | Attribute name   | Type   | Default Value | Description  |
-|:----------------:|:---------:|:------------:|:-------------:|
-| `root`             | String  | First body in the file | Changes the root body of the tree |
+|:----------------:|:------:|:------------:|:-------------:|
+| `root`           | String | First body in the file | Changes the root body of the tree |
+| `originXYZ`      |  List  | empty | Changes the position of the root body |
+| `originRPY`      |  List  | empty | Changes the orientation of the root body |
+
+##### Algorithm Parameters
+| Attribute name   | Type   | Default Value | Description  |
+|:----------------:|:------:|:-------------:|:-------------:|
+| `epsilon`        | Float | Machine *eps* | Set a custom value for testing whether a number is close to zero |
```

Comment:
Not a big problem, but the default value is four times the machine eps, see https://github.com/robotology/simmechanics-to-urdf/pull/35/files#diff-3ee04789adb44a425899309f3db7bf87R43 .

Reply:
Updated, thanks!